packages/shorebird_cli/lib/src/shorebird_env.dart

```diff
@@ -242,7 +242,10 @@ class ShorebirdEnv {
       platform.environment.containsKey('TF_BUILD');
 
   /// Adds `shorebird.yaml` to the `assets` section of the `pubspec.yaml` file
-  static void addShorebirdYamlToPubspecAssets(File pubspecFile) {
+  void get addShorebirdYamlToPubspecAssets {
```

Comment:
I'm not sure why this is a getter – it has side effects and should be a function

Reply:
Changed it to a function